packages/synapse-react-client/src/components/FullWidthAlert/FullWidthAlert.tsx

```diff
@@ -116,50 +106,104 @@ function FullWidthAlert(props: FullWidthAlertProps) {
       }
     }
   }, [onClose, autoCloseAfterDelayInSeconds])
-  const additionalAlertVariantClass = variant ? `alert-${variant}` : ''
-  return (
+
+  const gridSize = 12
+  const buttonGridSize = 4
+  const hasButtons = primaryButtonConfig || secondaryButtonConfig
+  const buttonGridSizeMap = {
+    sm: hasButtons ? buttonGridSize + 1 : 0,
+    md: hasButtons ? buttonGridSize - 1 : 0,
+    lg: hasButtons ? buttonGridSize : 0,
+  }
+
+  const alert = (
     <Alert
-      variant={variant}
-      show={show}
-      dismissible={false}
-      transition={transition}
-      className={`FullWidthAlert bootstrap-4-backport ${
-        isGlobal ? 'global' : ''
-      } ${additionalAlertVariantClass}`}
+      severity={variantToSeverity(variant)}
+      sx={{
+        width: '100%',
+        '.MuiAlert-message': {
+          flexGrow: 1,
+        },
+      }}
+      className={`FullWidthAlert ${isGlobal && 'global'}`}
+      onClose={onClose}
     >
-      <div className={`gridContainer ${onClose ? 'hasCloseButton' : ''}`}>
-        <span className="iconArea">{iconContent}</span>
-        <span className="mainContent">
-          <span className="messageArea">
-            <Typography variant="smallText2">{title}</Typography>
-            {title && description && <div className="SRC-marginBottomTen" />}
-            <Typography variant="smallText1">{description}</Typography>
-          </span>
-          <span className="buttonArea">
-            {secondaryButtonConfig && (
-              <ButtonFromConfig
-                className="secondaryButton"
-                config={secondaryButtonConfig}
-                variant="tertiary"
-              />
-            )}
-            {primaryButtonConfig && (
-              <ButtonFromConfig
-                className="primaryButton"
-                config={primaryButtonConfig}
-                variant="sds-primary"
-              />
-            )}
-          </span>
-        </span>
-        {onClose && (
-          <button className="closeAlert" onClick={onClose}>
-            <Clear fontSize={'large'} />
-          </button>
+      <Grid
+        container
+        rowSpacing={1}
+        columnSpacing={2}
+        direction={{ xs: 'column', sm: 'row' }}
```

Comment:
When screen is small, vertically stack the text and buttons. When screen is larger, move the text side-by-side with the buttons.

Reply:
main: 

https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/6012e7ab-3b8f-455b-9dd7-e71c8007b60f

feature: 

https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/c532f0fa-3dc1-4d42-87f4-cdcb6513c7b2

Reply:
Why was this changed from the current behavior? Not seeing an instance of this case in Figma

Reply:
Nvm, I see the other comment